.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://pre-commit.com for more information
```

Comment:
This is fine, but can you move this under your `battle` directory? That was we don't get merge conflicts from everyone else adding their `pre-commit` configs in their PRs.

Reply:
I tried that but if `.pre-commit-config.yaml` isn't in the "root" directory, it seems like `pre-commit` can't find the configuration file? When I have my config under the battle folder, `git commit` gives me this message:

```
No .pre-commit-config.yaml file was found
- To temporarily silence this, run `PRE_COMMIT_ALLOW_NO_CONFIG=1 git ...`
- To permanently silence this, install pre-commit with the --allow-missing-config option
- To uninstall pre-commit run `pre-commit uninstall`
```

Reply:
Ah right. My bad, I had forgotten that this is by design &mdash; when I was creating these Issues I was thinking of each of them being a separate project and forgetting that they would all need to have to contend with merge conflicts.

Uh, since I didn't about this well enough in advance just leave it how it is and then I'll give everyone else advice about how to just merge in the results of whatever PR gets merged first. So no need to do anything here. :+1:

Reply:
👍